native/src/modules/categories/components/Categories.js

```diff
@@ -132,6 +132,17 @@ class Categories extends React.Component<PropsType> {
     return categories.map(category => this.getListModel(category))
   }
 
+  getListContentModel (category: CategoryModel): ?ListContentModelType {
+    const { navigation, navigateToInternalLink, resourceCacheUrl } = this.props
+    return category.content ? {
```

Comment:
Is it possible that content is not defined? Or is this just for the root category?

Reply:
the inner categories do not always have a content, whereby content means Images, Text and stuff.

Reply:
Yeah, so it is a empty string. But does that matter?

Reply:
Ahhh. Now I get what you mean. Somehow I missed that the content is not actually optional. Yeah, I think this can be removed then.

Reply:
Or not. The html-lib doesn't want to render an empty string. I'll keep it and add a comment